src/Client.php

```diff
@@ -10,17 +10,8 @@
 
 /**
  * Client Interface
+ * @deprecated Client is replaced by ClientInterface and will be removed in 3.0.
```

Comment:
Should be noted since when it's deprecated - next minor - 2.9.0.

Reply:
Added since remark.